### ISSUE TITLE
About quantitative results in paper

### ISSUE THREAD

**Issue by jayhopark** [fudan-zvg/4d-gaussian-splatting#69]
Hi Yang,

Thank you for sharing this wonderful work!

I have a question regarding the quantitative results in your paper. I'm reproducing the result on the dnerf dataset, but my average PSNR is 32.29-lower than the 34.09 reported in Table 2.

[https://github.com/fudan-zvg/4d-gaussian-splatting/blob/3fe460f1f2342526bccf6c01b63da125faf0fa94/scene/dataset_readers.py#L315C1-L316C1](url)

I noticed that in the provided code, only the Lego scene uses the validation sets, while other dnerf scene use the test set during evaluation. 

Could you clarify which data split was used to calculate the reported PSNR? Was it computed on the validation set, the test set, or a combination of both?

Thanks in advance for your help!

![Image](https://github.com/user-attachments/assets/fcaf519a-3221-450d-bd15-60286b1311f7)

**Comment by Alexander0Yang** [comment]
Hi @jayhopark,
Thanks so much for your interest in our work, and very sorry for the late reply. 
I shared the test script and checkpoints for all scenes in the D-NeRF dataset in this [link](https://drive.google.com/file/d/1iiiKkuHox09_uz-UdT2CM_6URGG5MtYl/view?usp=sharing), hopes that can help your reproduction. Except for the `Lego` using the val split due to the well-known misalignment between its train and test set, all other scenes are evaluated on the test split. Due to the potential overfitting in some scenes, we stop the optimization in the early stage to avoid more floaters. The mean PSNR computed using these checkpoints is 34.07, almost the same as that obtained before our submission.

| Scene         | BouncingBalls | HellWarrior | Hook    | JumpingJacks | Lego    | Mutant  | StandUp | Trex    | **Mean**  |
|---------------|----------------|--------------|---------|----------------|---------|----------|----------|---------|-----------|
| PSNR     | 33.2897        | 34.9863      | 32.9605 | 33.3044       | 27.1388 | 39.4044 | 40.1556 | 31.2932 | **34.0666** |

Since these checkpoints were created before the modification of rotation matrix definition. Please check out commit 5094ca8 before running the test script.

**Comment by jayhopark** [comment]
Thank you for your kind response! @Alexander0Yang 

May I ask one more question?

I’m new to this field, so I didn’t fully understand the issue you mentioned:
“Except for the Lego using the val split due to the well-known misalignment between its train and test set, all other scenes are evaluated on the test split.”

Could you please explain this issue in more detail or provide some reference materials?

Thank you!
